airflow/providers/amazon/aws/hooks/cloud_formation.py

```diff
@@ -25,7 +25,7 @@
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
 
 
-class AWSCloudFormationHook(AwsBaseHook):
+class CloudFormationHook(AwsBaseHook):
```

Comment:
We are missing here also `class AWSCloudFormationHook(CloudFormationHook):` with deprecation warning

Reply:
Added